llpc/context/llpcPipelineContext.h

```diff
@@ -97,8 +103,9 @@ class PipelineContext {
 
   // Checks whether the pipeline is graphics or compute
   virtual bool isGraphics() const = 0;
-  // Check whether the pipeline uses features relevant to subgroup size
-  virtual bool usesSubgroupSize() const = 0;
+
+  // Gets subgroup size usage - bitmask with one bit per stage.
```

Comment:
This does not  say anything about the order of the bits. Is that important?

Reply:
It's not important for the users of the function right now, but I've extended the comment.